packages/webapp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "litefarm-webapp",
-  "version": "3.2.5",
+  "version": "3.2.6",
```

Comment:
also in conflict with: https://github.com/LiteFarmOrg/LiteFarm/pull/2618/files

Reply:
fixed thanks